index.html

```diff
@@ -46,7 +46,182 @@ <h1 class="title playfair" >Pritikant Dhal</h1>
   <div>
     <div class="bg"></div>
   </div>
- 
-  
+
+  <!--Project Section-->
+  <section id="projects" class="popins">
+  <h3 id="project-title" class="playfair title">projects</h3>
+  <article class="project flex">
+    <div>
+    <div class="bg1"></div>
+    </div> 
+  <p>Project name goes here</p>
+  <ul class="project-languages flex">
+    <li class="project-language">HTML/CSS</li>
+    <li class="project-language">Ruby on Rails</li>
+    <li class="project-language">JavaScript</li>
+  </ul>
+  <button class="project-btn flex">
+    <span>See this project</span>
+    <img
+       src="images/icons/ic_arrow_right.svg"
+       width="20.57px"
+       height="15.35px"
+       alt=""
+       srcset="" /> 
+  </button>
+  <div class="space">&nbsp;</div>
+</article>
+<article class="project flex">
+  <div>
+  <div class="bg2"></div>
+  </div> 
+<p>Project name goes here</p>
```

Comment:
- [ ] In order to adhere to [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and enhance code readability, it is recommended to use an appropriate heading tag for the project title, such as `<h3>, <h4>, or <h5>. `
- [ ] By employing the` <h>` tag and assigning the class name `"project-title"`, the code becomes more semantically meaningful and easier to understand.
- [ ] Additionally, it is beneficial to assign a meaningful class name to the element. Instead of `bg2` please use the appropriate one.